src/material-experimental/mdc-button/button-base.ts

```diff
@@ -75,30 +75,20 @@ export const _MatButtonBaseMixin: CanDisableRippleCtor&CanDisableCtor&CanColorCt
 
 /** Base class for all buttons.  */
 export class MatButtonBase extends _MatButtonBaseMixin implements CanDisable, CanColor,
-                                                                  CanDisableRipple, OnChanges {
-  rippleTarget: RippleTarget = {
-    rippleConfig: {
-      animation: {
-        // TODO(mmalerba): Use the MDC constants once they are exported separately from the
-        // foundation. Grabbing them off the foundation prevents the foundation class from being
-        // tree-shaken. There is an open PR for this:
-        // https://github.com/material-components/material-components-web/pull/4593
-        enterDuration: 225 /* MDCRippleFoundation.numbers.DEACTIVATION_TIMEOUT_MS */,
-        exitDuration: 150 /* MDCRippleFoundation.numbers.FG_DEACTIVATION_MS */,
-      },
-    },
-    rippleDisabled: false,
+                                                                  CanDisableRipple {
+  /** The ripple animation configuration to use for the buttons. */
+  _rippleAnimation: RippleAnimationConfig = {
+    enterDuration: MDCRippleFoundation.numbers.DEACTIVATION_TIMEOUT_MS,
```

Comment:
I think we still don't want to use these actually, according to the comment we're waiting for them to be exported separately from the foundation so we don't break tree shaking

Reply:
Looks like we're good, they export `numbers` apart from the foundation class